PostFreeSurfer/PostFreeSurferPipelineLongLauncher.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+    pipedirguessed=1
+    export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/processingmodecheck.shlib" "$@" # Check processing mode requirements
```

Comment:
This doesn't appear to be used - it exists to admonish the user if they try to use "legacy-data processing options" without acknowledging that we strongly discourage using them on modern data, and I don't see any of that type of logic here.

Reply:
ok